internal/tiltfile/local_resource.go

```diff
@@ -37,12 +41,22 @@ func (s *tiltfileState) localResource(thread *starlark.Thread, fn *starlark.Buil
 
 	deps := value.NewLocalPathListUnpacker(thread)
 
-	var resourceDepsVal starlark.Sequence
+	var resourceDepsVal, tagsVal starlark.Sequence
 	var ignoresVal starlark.Value
 	var allowParallel bool
 	var links links.LinkList
 	autoInit := true
 
+	var isTest bool
+	if fn.Name() == testN {
```

Comment:
I'm tempted to suggest separate functions to take care of everything through unpackArgs, and then a common helper function that deals with all that, so that we're not exposing all args to both functions (e.g. `test` shouldn't accept `serve_cmd` and `local_resource` shouldn't accept `tags`). Maybe it's annoying to list all those args out so many times though.

Reply:
wheeeee it's hack o'clock.

yes i agree, if we leave this code in long-term we should change this, but right now #yolo. i'll add a TODO.